app/controllers/alaveteli_pro/stripe_webhooks_controller.rb

```diff
@@ -60,10 +60,9 @@ def invoice_payment_succeeded
       subscription = Stripe::Subscription.retrieve(subscription_id)
       plan_name = subscription.plan.name
```

Comment:
Should this be plan.product.name (expanding product), if you're deciding to use that everywhere, to match the other change?

Reply:
Pushed [eb0f277](https://github.com/mysociety/alaveteli/pull/8405/commits/eb0f2770a2097d2157ed5d91146c74e4cb09e5e0) to change this. Annoyingly the Stripe mocking gem we're using doesn't "expand" resources so I've needed to stub this in some RSpec examples.